### PR TITLE
Romans to nums complete code 

### PR DESCRIPTION
**Pull request by nvega23**

### CODE REVIEW

File: romans_to_nums_solo.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+instances where subtraction is used:
+
+I can be placed before V (5) and X (10) to make 4 and 9.
+X can be placed before L (50) and C (100) to make 40 and 90.
+C can be placed before D (500) and M (1000) to make 400 and 900.
+Given a roman numeral, convert it to an integer. """
+
+
+def romanToInt(input_string: str) -> int:
+	romans = {
```
Comment (kswang2400) [inline romans_to_nums_solo.py:14]: nit: indents look off, make sure its 4 spaces?

File: romans_to_nums_solo.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		"M": 1000
+	}
+	last_character = None
+	output = 0
+	for char in input_string:
+		value = romans[char]
+		output += value
+		if last_character:
+			if last_character == "I" and char in ["V", "X"]:
+				output -= 2
```
Comment (kswang2400) [inline romans_to_nums_solo.py]: nit: so generally "magic numbers" are harder to understand and maintain. what you could do here to be more expressive is say 

```
output -= 2 * roman['I']
output -= 2 * roman['X']
output -= 2 * roman['C']
```

so when you come back you know why you're subtracting 2, 20, and 200

**Review by kswang2400 — APPROVED** [review]
nice :)

File: romans_to_nums_solo.py (hunk 1 above)
Comment (nvega23) [reply romans_to_nums_solo.py:14]: The indents in sublime are set to 4 but the file looks different on here than on sublime

**Pull request merged**